community_server/src/Model/Table/TransactionsTable.php

```diff
@@ -194,7 +194,7 @@ public function listTransactionsHumanReadable($stateUserTransactions, array $use
                     
                     if($balance > 100) 
                     {
-                      $final_transactions['decay'] = [ 
```

Comment:
This was the error we have seen in the notice?

Reply:
Yes this was the error.
Instead of putting the decay in the final_transaction the transaction current constructed in the loop, I put it into final_transactions which contain all the transactions
```php 
$final_transactions = [$final_transaction];
````
How should that be detected automatic?
That's php code not C/C++.  In C/C++ that couldn't never happen.